lib/goldencobra/version.rb

```diff
@@ -1,3 +1,3 @@
 module Goldencobra
-  VERSION = "2.3.3".freeze
+  VERSION = "3.0.0".freeze
```

Comment:
Finde eigentlich nicht, dass das ein so großer Sprung ist. `2.4.0` reicht doch, oder? Wirklich verändert sich ja nichts. Nur die Railsversion und dahingehend Anpassungen. Sonst aber nichts bahnbrechend in der Nutzung oder im Aussehen oder beim Setup oder Sonstiges.